src/v2/utils/calcChanges.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+export default (oldValue: string | number, newValue: string | number) => {
```

Comment:
this needs to be a server-side function (or client-side function that uses the API differently because the desired data isn't present right now)